devicetypes/smartthings/ecobee-thermostat.src/ecobee-thermostat.groovy

```diff
@@ -32,7 +32,7 @@ metadata {
 		command "switchMode"
 		command "switchFanMode"
 
-		attribute "thermostatSetpoint", "number"
```

Comment:
This attribute, "thermostatSetpoint" is part of the Thermostat capability and thus doesn't need to be declared by the DTH. SA can use it and the potential issue with existing SA is if they used it and expect the invalid values "Auto" and "Off" to be used. If they are they don't comply with the current model, however if the still need this they can use the new custom attribute instead.